auth/auth_test.go

```diff
@@ -72,6 +72,7 @@ func TestSignatureTxMiddlewareMultipleTxSameBlock(t *testing.T) {
 	//State is reset on every run
 	ctx2 := context.WithValue(context.Background(), ContextKeyOrigin, origin)
 	state2 := loomchain.NewStoreState(ctx2, store.NewMemStore(), abci.Header{Height: 27}, nil, nil)
+	//nolint: ineffassign
```

Comment:
Don't ignore these warnings, fix them.

Reply:
Actually assignment was assigning some variables which might be useful later on in tests but were not currently being used. I have fixed it as they can be assigned if needed.